community-extra-vars.yml

```diff
@@ -1,18 +1,34 @@
 acs:
   artifact_name: alfresco-content-services-community-distribution
   repository: "{{ nexus_repository.releases }}"
-  version: 23.1.0-A45
+  version: 23.1.0
   edition: Community
+api_explorer:
+  artifact_name: api-explorer
+  repository: "{{ nexus_repository.releases }}"
+  version: 23.1.0
+transform:
+  artifact_name: alfresco-transform-core-aio
+  repository: "{{ nexus_repository.releases }}"
+  version: 5.0.0
+acc:
+  artifact_name: alfresco-control-center
+  repository: "{{ nexus_repository.releases }}"
+  version: 8.3.0
+search:
+  artifact_name: alfresco-search-services
+  repository: "{{ nexus_repository.releases }}"
+  version: 2.0.8.2
```

Comment:
why?

Reply:
mostly to not break the updatecli pipeline for community